.goreleaser.yml

```diff
@@ -0,0 +1,27 @@
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    #- go mod download
+    # you may remove this if you don't need go generate
+    #- go generate ./...
+    - make generate
+    - git status
+builds:
+- env:
+  - CGO_ENABLED=0
+  binary: manager
+  goos:
+  - linux
+  goarch:
+  - amd64
```

Comment:
Only `amd` supported?

Reply:
harbor doesn't support other arch now, so there's no need to release for others too